edx-platform/pearson-theme/lms/templates/footer.html

```diff
@@ -109,10 +109,10 @@
       % if not hide_openedx_link:
       <div class="footer-about-openedx">
         <p>
-            <a href="http://open.edx.org">
```

Comment:
@Adonisrq94 please change this back to "http://open.edx.org'. the logos are separate and the links must reflect the logo, we can navigate to proversity.org on the EDX.org logo.